src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
```

Comment:
Only one line should separate import statements and top-level class declarations. So please remove one of these blank lines.
Ref: https://mate-academy.github.io/style-guides/java/java.html#s3-source-file-structure

Reply:
And also please remove multiple blank lines for whole file.